modules/extensions/src/mask/mask-effect.ts

```diff
@@ -56,7 +56,14 @@ export default class MaskEffect implements Effect {
       });
     }
 
-    const maskLayers = layers.filter(l => l.props.visible && l.props.operation === OPERATION.MASK);
+    if (pass.includes('picking')) {
```

Comment:
Seems like this should be called `passes`...

Reply:
This method is called once per render pass. `pass` is the name of the render pass, and `picking-*` is the naming convention set in DeckPicker.

Reply:
Aha. I usually use `.includes()` with arrays, not strings, so I assumed this was an array when I saw `.includes`. 
It is a total nit, obviously, but I think `.startsWith()` would have avoided leading me astray here.